mempool/src/pool/store.rs

```diff
@@ -123,6 +123,32 @@ impl MempoolStore {
         self.txs_by_id.get(&id.get())
     }
 
+    pub(super) fn assert_valid(&self) {
```

Comment:
I think it would be nice to give this function a name that emphasizes that it only performs some checks when the `test` feature is defined, though I cannot suggest a better name.

Reply:
I see what you're saying, just can't seem to think of a name either... Something like `assert_valid_test_only` would make me cringe :grimacing: